docs/source/reference/northstar/modsettings.rst

```diff
@@ -0,0 +1,117 @@
+Mod Settings
+====
+
+Setup
+----
+
+ConVars are the easiest way to implement settings for your mod using the Mod Settings API.
+
+Your mod needs to register itself and all ConVars that are a part of your mod that should be accessible in the Mod Settings menu. To do this, simply add a new script to your mod that runs only in the UI VM like this:
+
+.. code-block:: json
+
+  "Path": "ui/ms_example_mod.nut",
+  "RunOn": "UI",
+  "UICallback": {
+    "Before": "ExampleMod_AddModSettings"
+  }
+    
+Inside of the callback specified here, you can add your settings.
+
+API
+----
+
+.. warning::
+
+  ConVar values will only persist if the ConVar has an ARCHIVE flag. For Clients, use ``FCVAR_ARCHIVE_PLAYERPROFILE``.
+  
+  All Mod Settings functions have a ``stackPos`` paramter. This parameter should only be changed if you're writing custom wrappers for the settings.
+
+.. cpp:function:: void AddModTitle( string modName, int stackPos = 2 )
+
+  Adds a new category in the settings for your mod
+  
+  .. note::
+  
+    It's mandatory to register a mod before you can add any settings
+    
+.. cpp:function:: void AddModCategory( string categoryName )
+
+  Adds a new category to your mod
+  
+  .. note::
+  
+    It's mandatory to register a category for your mod. A mod may have multiple categories
+    
+.. cpp:function:: void AddConVarSetting( string conVar, string displayName, string type = "", int stackPos = 2 )
+
+  Adds a basic setting to the last declared category.
+
+  **Parameters:**
+  
+  * ``string conVar`` - the conVar this setting modifies
```

Comment:
typo in the upper/lower case of `conVar` vs `ConVar`?

(The description. not the parameter)